example/storybook/src/components/Forms/Checkbox/index.stories.mdx

```diff
@@ -461,30 +461,26 @@ The Examples section provides visual representations of the different variants o
     showArgsController={false}
     metaData={{
       code: `
-<VStack space="3xl">
- <Checkbox justifyContent='left' defaultIsChecked size="lg">
-  <Checkbox.Indicator bgColor="$darkBlue600" >
- <Icon as={CheckIcon} color="$backgroundLight50"/>
-   </Checkbox.Indicator>
-  <Checkbox.Label>Framer</Checkbox.Label>
-</Checkbox>
-<Checkbox justifyContent='left' size="lg">
-  <Checkbox.Indicator>
-    <Checkbox.Icon>
-      <Icon as={CheckIcon} />
-    </Checkbox.Icon>
-  </Checkbox.Indicator>
-  <Checkbox.Label>Invision Studio</Checkbox.Label>
-</Checkbox>
-<Checkbox justifyContent='left' size="lg">
-  <Checkbox.Indicator>
-    <Checkbox.Icon>
-      <Icon as={CheckIcon} />
-    </Checkbox.Icon>
-  </Checkbox.Indicator>
-  <Checkbox.Label>Adobe XD</Checkbox.Label>
-</Checkbox>
-</VStack>
+          <VStack space="3xl">
```

Comment:
please correct the indentation in this example

Reply:
done